src/overlays/actors/ovl_En_Nb/z_en_nb.c

```diff
@@ -21,255 +169,1401 @@ const ActorInit En_Nb_InitVars = {
+    if (animFinished) {
+        EnNb_SetCurrentAnim(this, &D_06005614, 0, 0.0f, 0);
+    }
+}
+
+void EnNb_CheckCreditsCsModeImpl(EnNb* this, GlobalContext* globalCtx) {
+    CsCmdActorAction* csCmdNPCAction = EnNb_GetNpcCsAction(globalCtx, 1);
+    s32 action;
+    s32 unk_28C;
+
+    if (csCmdNPCAction != NULL) {
+        action = csCmdNPCAction->action;
+        unk_28C = this->unk_28C;
+        if (action != unk_28C) {
+            switch (action) {
+                case 15:
+                    EnNb_SetupCreditsFadeIn(this, globalCtx);
+                    break;
+                case 16:
+                    EnNb_SetupCreditsHeadTurn(this);
+                    break;
+                default:
+                    // "En_Nb_inEnding_Check_DemoMode: OPERATION DOESN'T EXIST!!!!!!!!"
+                    osSyncPrintf("En_Nb_inEnding_Check_DemoMode:そんな動作は無い!!!!!!!!\n");
+                    break;
+            }
+            this->unk_28C = action;
+        }
+    }
+}
+
+void EnNb_CheckCreditsCsMode(EnNb* this, GlobalContext* globalCtx) {
+    EnNb_CheckCreditsCsModeImpl(this, globalCtx);
+}
+
+void EnNb_CreditsFade(EnNb* this, GlobalContext* globalCtx) {
+    func_80AB1284(this, globalCtx);
+    EnNb_FrameUpdateMatrix(this);
+    EnNb_UpdateEyes(this);
+    EnNb_SetAlphaInCredits(this);
+    EnNb_SetupCreditsSit(this);
+}
+
+void func_80AB3428(EnNb* this, GlobalContext* globalCtx) {
+    func_80AB1284(this, globalCtx);
+    EnNb_FrameUpdateMatrix(this);
+    EnNb_UpdateEyes(this);
+    EnNb_CheckCreditsCsModeImpl(this, globalCtx);
+}
+
+void EnNb_LookUp(EnNb* this, GlobalContext* globalCtx) {
+    s32 animFinished;
+
+    func_80AB1284(this, globalCtx);
+    animFinished = EnNb_FrameUpdateMatrix(this);
+    EnNb_UpdateEyes(this);
+    EnNb_CheckIfLookingUp(this, animFinished);
+}
+
+void EnNb_CheckToSpawnNearSpiritCrawlspace(EnNb* this, GlobalContext* globalCtx) {
+    if (!(gSaveContext.eventChkInf[9] & 0x20) && gSaveContext.linkAge == 1) {
+        EnNb_GetPathInfo(this, globalCtx);
+        if (!(gSaveContext.eventChkInf[9] & 0x10)) { // looking into crawlspace
+            EnNb_SetCurrentAnim(this, &D_06006E78, 0, 0.0f, 0);
+            this->action = NB_CROUCH_CRAWLSPACE;
+            this->drawMode = NB_DRAW_DEFAULT;
+        } else {
+            EnNb_SetCurrentAnim(this, &D_06004BB4, 0, 0.0f, 0);
+            this->headTurnFlag = 1;
+            this->actor.flags |= 9;
+            this->actor.posRot.pos = this->finalPos;
+            this->action = NB_IDLE_AFTER_TALK;
+            this->drawMode = NB_DRAW_DEFAULT;
+        }
+    } else {
+        Actor_Kill(&this->actor);
+    }
+}
+
+void func_80AB359C(EnNb* this) {
+    PosRot* posRot = &this->actor.posRot;
+    Vec3f* initialPos = &this->initialPos;
+    Vec3f* finalPos = &this->finalPos;
+    f32 f0;
+    u16 temp_t1;
+    s16 temp_2;
+
+    this->movementTimer++;
+    temp_2 = kREG(17);
+    temp_t1 = temp_2;
+    temp_t1 += 0x19; // Oddly needs all these temps to match
```

Comment:
```suggestion
    temp_t1 += 25; // Oddly needs all these temps to match
```
timers in decimal makes more sense imo (also with same kREG in 1+ other place)